src/cascadia/TerminalApp/SuggestionsControl.xaml

```diff
@@ -103,23 +103,10 @@
         </ResourceDictionary>
     </UserControl.Resources>
 
-    <Grid>

```

Comment:
guess what? we basically weren't using this `Grid` for anything. There was only ever `_backdrop` in it, taking up the whole thing. 

This was a side-effect of command palette copypasta